<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-N2O8J3T" failures="2" tests="9" name="Test" time="0.186" errors="0" timestamp="2024-08-09T10:57:08 IST">
  <testcase classname="listeners.AnalyzerDemo" name="test1" time="0.022"/>
  <testcase classname="listeners.AnalyzerDemo" name="test2" time="0.013">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="listeners.AnalyzerDemo" name="test2" time="0.003">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="listeners.AnalyzerDemo" name="test2" time="0.003">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="listeners.AnalyzerDemo" name="test2" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at listeners.AnalyzerDemo.test2(AnalyzerDemo.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="listeners.AnalyzerDemo" name="test3" time="0.004">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase classname="listeners.AnalyzerDemo" name="test3" time="0.003">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase classname="listeners.AnalyzerDemo" name="test3" time="0.004">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase classname="listeners.AnalyzerDemo" name="test3" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at listeners.AnalyzerDemo.test3(AnalyzerDemo.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
</testsuite> <!-- Test -->
